src/main/webapp/WEB-INF/appengine-web.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>false</threadsafe>
+  <sessions-enabled>true</sessions-enabled>
+  <runtime>java8</runtime>
+  <static-files>
+    <!-- prevent unwanted caching when accessing via the web preview server -->
+    <include path="/**" expiration="0s" />
```

Comment:
Does this only affect the preview server or will it affect prod as well? Not like it matters here, just curious.

Reply:
Not really sure how this works. This was a boilerplate given to us in walk-through.

Reply:
Interesting, guess it doesn't matter.